templates/keycloak.service.erb

```diff
@@ -4,10 +4,12 @@ After=network.target
 
 [Service]
 Type=idle
-#Environment="JAVA_OPTS=-Xms1024m -Xmx20480m -XX:MaxPermSize=768m"
 <% if scope['keycloak::service_environment_file'] -%>
 EnvironmentFile=<%= scope['keycloak::service_environment_file'] %>
 <% end -%>
+<% if scope['keycloak::service_java_opts'] -%>
+Environment="JAVA_OPTS=<%= scope['keycloak::service_java_opts'].join(' ') %>"
```

Comment:
Need to check if `String` vs `Array` and only use `join` for `Array`:

```
2.4.0 :001 > '-Xmx=1024m'.join(' ')
NoMethodError: undefined method `join' for "-Xmx=1024m":String
```

Could probably do second condition inside current `if` that is `if scope['keycloak::server_java_opts'].is_a?(Array)` and the `else` condition would be for `String`.